v2/pkg/reporting/trackers/jira/jira.go

```diff
@@ -91,6 +94,52 @@ func (i *Integration) CreateIssue(event *output.ResultEvent) error {
 	return nil
 }
 
+// CreateIssue creates an issue in the tracker or updates the existing one
+func (i *Integration) CreateIssue(event *output.ResultEvent) error {
+	if i.options.UpdateExisting {
+		issue_id, err := i.FindExistingIssue(event)
```

Comment:
please follow the linter guidelines on variable naming (camelCase instead of snake_case)